componentLibrary.css

```diff
@@ -497,4 +497,99 @@
     background-color: black;
     margin: 10px;
     color: white;
-}
\ No newline at end of file
+}
+
+/* responsive image starts from here */
+.img--responsive {
+    width: 100%;
+}
+
+.reponsive__image--contianer {
+    width: 12rem;
+}
+
+.img--rounded {
+    border-radius: 100%;
+}
+
+/* toast start from here */
+
+#snackbar {
+    visibility: hidden;
+    min-width: 200px;
+    color: white;
+    text-align: center;
+    border-radius: 1rem;
+    padding: 1rem;
+    background-color: green;
+    position: fixed;
+    bottom: 30px;
+    left: 50%;
+    margin-left: -100px;
+    z-index: 1;
+}
+
+#snackbar.show {
+    visibility: visible;
+    -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
+    animation: fadein 0.5s, fadeout 0.5s 2.5s;
+}
+
+@-webkit-keyframes fadein {
+    from {bottom: 0; opacity: 0;}
+    to {bottom: 30px; opacity: 1;}
+}
+@keyframes fadein {
+    from {bottom: 0; opacity: 0;}
+    to {bottom: 30px; opacity: 1;}
+}
+@-webkit-keyframes fadeout {
+    from {bottom: 30px; opacity: 1;}
+    to {bottom: 0; opacity: 0;}
+}
+@keyframes fadeout {
+    from {bottom: 30px; opacity: 1;}
+    to {bottom: 0; opacity: 0;}
+}
+
+/* text utilities start here */
+
+.heading-container {
+    width: 70%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    color: white;
+}
+
+.heading--one {
+    font-size: 48px;
```

Comment:
Please use rem

Reply:
oh sorry, I will change it to rem. Thank you.